packer/windows-vhd-builder.json

```diff
@@ -0,0 +1,67 @@
+{
+    "variables": {
+        "client_id": "{{env `AZURE_CLIENT_ID`}}",
+        "client_secret": "{{env `AZURE_CLIENT_SECRET`}}",
+        "tenant_id": "{{env `AZURE_TENANT_ID`}}",
+        "subscription_id": "{{env `AZURE_SUBSCRIPTION_ID`}}"
+    },
+    "builders": [
+        {
+            "type": "azure-arm",
+            "client_id": "{{user `client_id`}}",
+            "client_secret": "{{user `client_secret`}}",
+            "tenant_id": "{{user `tenant_id`}}",
+            "subscription_id": "{{user `subscription_id`}}",
+            "location": "westus2",
+            "vm_size": "Standard_D4s_v3",
+            "os_type": "Windows",
+            "image_publisher": "MicrosoftWindowsServer",
+            "image_offer": "WindowsServer",
+            "image_sku": "2019-Datacenter-Core-smalldisk",
+            "image_version": "17763.615.1907121548",
+            "managed_image_resource_group_name": "packer",
+            "managed_image_name": "aksengine-windows-{{timestamp}}",
+            "managed_image_storage_account_type": "Premium_LRS",
+            "communicator": "winrm",
+            "winrm_use_ssl": true,
+            "winrm_insecure": true,
+            "winrm_timeout": "10m",
+            "winrm_username": "packer"
+        }
+    ],
+    "provisioners": [
+        {
+            "elevated_user": "packer",
+            "elevated_password": "{{.WinRMPassword}}",
+            "environment_vars": "ProvisioningPhase=1",
+            "type": "powershell",
+            "script": "packer/configure-windows-vhd.ps1"
+        },
+        {
+            "restart_timeout": "10m",
+            "type": "windows-restart"
+        },
+        {

```

Comment:
This second reboot is added for reliability because after the first reboot Hyper-V messes with the network adapters causing intermittent issues with the WinRM communicator.
After adding a second reboot here all the flakiness went away.
I'm looking for a good way to document that here/am open to suggestions but json doesn't all comments :(